honeycomb-kernels/Cargo.toml

```diff
@@ -15,3 +15,4 @@ publish = true
 [dependencies]
 honeycomb-core = { workspace = true, features = ["io", "utils"] }
 vtkio.workspace = true
+num.workspace = true
```

Comment:
What is this workspace?

Reply:
The actual version of dependencies are defined in the root manifest (i.e. the workspace's); This is basically telling cargo to use the version specfied in the workspace, that way there is only one `num` dep to update even if multiple member use the crate.